tedana/workflows/post_taskcorr.py

```diff
@@ -0,0 +1,259 @@
+"""
+Quick hack for tedana, from HBM Hackathon 2019.
+TODO:
+- let user decide wether they want NCC or simple correlation.
+- Better threshold.
+
+"""
+
+import logging
+
+import sys
+import argparse
+
+import numpy as np
+import pandas as pd
+
+import os.path as op
+import scipy.stats as sct
+
+from tedana.workflows.parser_utils import is_valid_file
+
+LGR = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def _get_parser():
+    """
+    Parses command line inputs for this function
+
+    Returns
+    -------
+    parser.parse_args() : argparse dict
+
+    """
+    parser = argparse.ArgumentParser()
+    # Argument parser follow template provided by RalphyZ, also used by tedana.
+    # https://stackoverflow.com/a/43456577
+    optional = parser._action_groups.pop()
+    required = parser.add_argument_group('required arguments')
+    required.add_argument('-td', '--ted-dir',
```

Comment:
Let's just call this `--dir` and `-d` to make things a little more straightforward.